web/templates/shared/flashes.html.eex

```diff
@@ -0,0 +1,9 @@
+<%= for flash_key <- [:info, :error] do %>
+  <%= if flash = get_flash(@conn, flash_key) do %>
+    <div class="flash flash-<%= flash_key %>">
+      <div class="container">
+        <p data-role="flash"><%= raw flash %></p>
```

Comment:
Why is `raw` necessary? Probably harmless, but if you ever included data from the database in a flash message this then this it might be an issue